doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,351 @@
+
+
+For additional command line arguments, see the *Notes* section in the
+description for the :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>`
+function.
+
+You can use a script file (batch ``'.bat'`` or PowerShell ``'.ps'``)
+to run the preceding commands all at once.
+
+Once you have launched MAPDL, you should see the following content
+in your console (or the output file):
+
+.. code::
+
+    Start GRPC Server
+
+    ##############################
+    ### START GRPC SERVER      ###
+    ##############################
+
+    Server Executable   : MapdlGrpc Server
+    Server listening on : 0.0.0.0:50052
+
+
+.. warning:: 
+  
+   Notice that the port specified in the console is the internal Docker container port.
+   This port has been mapped to the value specified for the :envvar:`LOCAL_MAPDL_PORT`
+   environment variable.
+
+
+.. _run_an_mapdl_image_using_docker_compose:
+
+Using ``docker-compose`` to launch MAPDL
+----------------------------------------
+
+You can also use `docker-compose <docker_compose_>`_ command to launch MAPDL configured in
+a `docker-compose <docker_compose_>`_ file.
+This is useful if you want to load an already configured environment, or
+if you want to launch multiple instances of MAPDL or services.
+
+For you convenience, the directory `docker <pymapdl_docker_dir_>`_ 
+contains pre-configured `docker-compose <docker_compose_>`_ files that you can
+use.
+It is recommended to use this `docker-compose.yml <pymapdl_docker_compose_base_>`_ file.
+This is the **base** configuration file which launches an instance of MAPDL which you
+can connect to as remote.
+
+
+.. _pymapdl_connect_to_MAPDL_container:
+
+Connect to the MAPDL container from Python
+==========================================
+
+You can connect to the MAPDL instance following :ref:`connect_grpc_madpl_session`.
+You do not need to specify an IP since Docker maps the ports to the local host.
+
+
+Additional considerations
+=========================
+
+Use ``--restart`` policy with MAPDL products
+--------------------------------------------
+
+By default, MAPDL creates a ``LOCK`` file in the working directory when it starts
+and deletes this file if it exits normally. The file is used to avoid overwriting files
+such as database (DB) files or result (RST) files when starting MAPDL after an
+abnormal termination.
+
+Because of this behavior, when using the Docker ``--restart`` flag in the `docker run <docker_run_>`_ 
+command, you might enter into an infinite loop after crashing if you specify the Docker image to
+reboot after an abnormal termination.
+When there is an abnormal termination (MAPDL crashes) the file :file:`LOCK` is kept on the
+working directory. Since MAPDL has exited, the container also exit.
+This trigger Docker ``restart`` policy, which attempts to restart MAPDL container and the MAPDL
+process with it. 
```

Comment:
```suggestion
When there is an abnormal termination (MAPDL crashes), the :file:`LOCK` file is kept on the
working directory. Since MAPDL has exited, the container also exits.

This triggers the Docker ``restart`` policy, which attempts to restart MAPDL container and the MAPDL
process with it.
```